### PR TITLE
[Translation it] performance/rendering/stick-to-compositor-only-properties-and-manage-layer-count.md

### PR DESCRIPTION
**Pull request by lucab85**
Italian translation of [performance/rendering/stick-to-compositor-only-properties-and-manage-layer-count.md](https://developers.google.com/web/fundamentals/performance/rendering/stick-to-compositor-only-properties-and-manage-layer-count).
Please review by @carlaofarrell

[Check out a review request at GitLocalize](https://gitlocalize.com/repo/107/it/review/396)

### CODE REVIEW

**Comment by petele** [comment]
@agektmr - any idea why we're getting a NO on the CLA?

**Comment by agektmr** [comment]
@petele this is a known issue on GitLocalize side. They should have changed signature behavior for machine translated commits. But apparently it's not activated yet cc: @sotayamashita @chikathreesix

**Comment by lucab85** [comment]
Filed a [bug](https://github.com/gitlocalize/feedback/issues/30): the CLA:NO is because "machine translation" commit with "mt@gitlocalize.com".

File: src/content/it/fundamentals/performance/rendering/stick-to-compositor-only-properties-and-manage-layer-count.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
+project_path: /web/fundamentals/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: La composizione è il momento in cui vengono raggruppate le parti
```
Comment (petele) [inline src/content/it/fundamentals/performance/rendering/stick-to-compositor-only-properties-and-manage-layer-count.md]: description shouldn't break lines.

File: src/content/it/fundamentals/performance/rendering/stick-to-compositor-only-properties-and-manage-layer-count.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+<img
+src="images/stick-to-compositor-only-properties-and-manage-layer-count/layer-view.jpg"
+alt="The layer view in Chrome DevTools.">
+
+Usando questa vista puoi tenere traccia del numero di livelli che hai. Se stai
+spendendo molto tempo nel compositing durante azioni critiche per le prestazioni
+come lo scorrimento o le transizioni (dovresti mirare a circa **4-5 ms** ). Puoi
+usare queste informazioni per vedere quanti livelli sono in uso e perché sono
+stati creati e da lì gestire il conteggio dei livelli della tua app.
```
Comment (petele) [inline src/content/it/fundamentals/performance/rendering/stick-to-compositor-only-properties-and-manage-layer-count.md]: Per our [style guide](https://developers.google.com/web/resources/translations#credit), please put the translated by at the bottom of the article.

**Comment by lucab85** [comment]
Done and rebased

**Pull request merged**